src/algebra/archimedean.lean

```diff
@@ -82,6 +86,17 @@ let ⟨M, hM⟩ := pow_unbounded_of_one_lt x hy in
 let ⟨n, hn₁, hn₂⟩ := int.exists_greatest_of_bdd hb he in
   ⟨n, hn₁, lt_of_not_ge (λ hge, not_le_of_gt (int.lt_succ _) (hn₂ _ hge))⟩
 
+/-- The same as exists_int_pow_near, but with ≤ and < the other way around. -/
```

Comment:
This docstring shows up whenever someone hovers over `exists_int_pow_near'` with their mouse in VScode (in an arbitrary file). But then your sentence seems a bit out of context. It would hence be better if you could write specialised docstrings for each declaration that make sense outside the context of this file.

Reply:
edited docstrings. each lemmas also now has a comment that points at the other.